src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/telemetry/telemetry.component.html

```diff
@@ -132,6 +132,28 @@
                 </div>
               </div>
             </div>
+
+            <!-- Channel perf -->
+            <div class="form-group row">
+              <label class="cd-col-form-label"
+                     for="channel_perf">
+                <ng-container i18n>Perf</ng-container>
```

Comment:
Just an observation: [`ng-container` ](https://angular.io/api/core/ng-container) is meant to hold directives (like `*ngIf`, `*ngTemplateOutlet`...) otherwise it's better to just use `<span i18n>` or equivalent.

Reply:
Ah, good to know. I'll see if the dashboard tests pass this time, and if not, I'll try switching this out.

Reply:
@alfonsomthd before I switch out `ng-container`, is there a way for me to check the results of https://eyes.applitools.com/app/batches/00000251765651500563/00000251765651500282?accountId=hCTedwnSTUWSCZlXPqua0g~~? I don't have access to the Applitools dashboard group.

Reply:
@ljflores If you click on [ceph dashboard tests details](https://jenkins.ceph.com/job/ceph-dashboard-pull-requests/9244/)
and go to Console Output, you'll see a line like this:
```Recorded Run: https://dashboard.cypress.io/projects/k7ab29/runs/5810```.
You should be able to enter the Cypress Dashboard and check the details.

Reply:
Thanks!